### DISCUSSION TITLE
Empirical research and evidence for spiral dynamics stages (amber, orange, green, teal etc)

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#444] [Research]
**Now moved to https://forum.secondrenaissance.net/t/empirical-research-and-evidence-for-spiral-dynamics-stages-amber-orange-green-teal-etc/140**

Does anyone know know of any empirical research and evidence for the famous spiral dynamic color stages? For example things like:

- Evaluation / questionnaire that is linked to stage assessment
- Published research and/or data from doing such questionnaire
- Information on reliability (crudely: same results in repetition)
- Information on accuracy (if that could be assessed)

I've struggled to find this kind of stuff ...

## Tasks

- [ ] Reference / excerpt from spiral dynamics book
- [ ] Connect with usage in Integral via Wilber (who starts using this from the 90s)
- [ ] Add examples of claims / references to research / evidence from e.g. Wilber (ie. to demonstrate that there is believed to be evidence)
- [ ] Online search for spiral dynamics data library online etc

**Comment by baouroux** [comment]
Great questions, and a topic of long term interest in the developmental community, even if the data is still sparse and lacking 'integration' between the various branches of what's being measured (e.g. values vs. world-view vs. wisdom vs. ego vs. morals vs. cognitive complexity).

I've haven't done the Spiral Dynamics training, but met Don Beck in 2008 and have followed fairly closely the use of the model in organisational development consulting. People like Henry Andrews have stood out as really studying the original research by Clare Graves, and using the original coding schema - before the simplifications with the colour coding schema that Beck and Cowan offered. Henry Andrews gives a [very helpful overview of the Graves' original work here](https://hellametamodernism.com/but-the-data/).

Graves called his theory in the end the emergent cyclical levels of existence (E-C theory or ECLET), and the [Wikipedia page](https://en.wikipedia.org/wiki/Graves%27s_emergent_cyclical_levels_of_existence?ref=hellametamodernism.com#Motivation_and_experimental_design) is pretty detailed.... there are a couple of [dissertations](https://digital.library.unt.edu/ark:/67531/metadc331022/) that came up when Googling for reliability and validity of the Graves model, and this one more specifically for the [Spiral Dynamics colours based on a values acceptance/rejection test](https://www.cognadev.com/Cognadev_Technical_Report_2_VO_Retest_20_Aug_14.pdf).

Still lacking though research at scale.

I wonder if there has been any attempt to link larger surveys such as the World Values Surveys with Spiral Dynamics? https://www.worldvaluessurvey.org/WVSContents.jsp

Jumping ahead to the more recent models and developmental scales, here is a good positioning paper of a more recent testing model 'STAGES', that give a good background to the key origins of developmental measurement, and [picks up on reliability and validity/accuracy, by Tom Murray](https://drive.google.com/file/d/11rJVWkD4lwPTbBMhzPhMchfBWpXoAvVC/view?usp=sharing).

Also relevant to your other thread announcement and question on the Lectica training (I'll post this link to that too) is this report by Jonathan Reams on the Foundations of Lectica Assessment course. I've worked closely with Jonathan on Sensemaking narratives projects, and we took the Lectica certification at the same time. He gives also a good overview of the publications by Theo Dawson et al. on the Lectica assessments reliability and validity - I can pick out some of them if of interest to deeper into Lectica.

One comment on Lectica, is that it had for me the most convincing reliability and validity evidence, BUT it is also a tool that really zooms into the cognitive complexity line of development, at the sacrifice of capturing a person's values. There is a paper on how you can score on very high cognitive complexity (i.e. levels of abstractions and systems representations) when presentating very immoral world views.

My case here is for not just more '3rd person' data , but that the developmental stages become a 2nd person grammar for sensemaking - I'm afraid you'll hear me come back to this a few times!

PS Henry Andrews has done a great job in assembling all the threads of the recent debates about the use of developmental stages: https://hellametamodernism.com/debate/

**Reply by rufuspollock** [reply]
Amazing response - thank-you @baouroux. 

I note one point from the excellent: https://hellametamodernism.com/but-the-data/ (your first link):

> But there's one big problem:  The data is long gone.
> 
>
> Wait... no data?
>
> For whatever reason, **towards the end of his life Graves threw out all but a few example essays, along with any notes from his judges that may have existed.** It's possible that some of the material is stuffed in a box in an archive somewhere at Union College, but unless and until someone finds it, that doesn't do us much good.
> 
> There's no real doubt that the data existed, as a number of people saw it, and Graves published a peer-reviewed paper outlining the theory based on it, and presented additional collated results at some mainstream conferences.  But no one analyzed it independently in sufficient detail to verify whether the same theory would emerge for them, either by re-judging the essays or by examining the output of the original judges.
> 
> The other possible source of data would be from Spiral Dynamics practitioners, **but while I have heard some statistics from such data, I am not aware of any publication of a sufficiently large data set along with the tools and methodology used to produce it in sufficient detail to allow for independent verification.** [emphasis added]

**Comment by rufuspollock** [comment]
x-posted to https://forum.secondrenaissance.net/t/empirical-research-and-evidence-for-spiral-dynamics-stages-amber-orange-green-teal-etc/140
